bin/cron/update_pledge_map

```diff
@@ -26,34 +26,60 @@ CENSUS_FORM_KINDS = "'Census2017','HocCensus2017'".freeze
 file = Tempfile.new(['data', '.csv'])
 begin
   CSV.open(file.path, 'w') do |csv|
+    DASHBOARD_DB[:census_submissions].
+      select_append(:census_submissions__id___census_submissions_id).
+      join(:census_submissions_school_infos, census_submission_id: :id).
+      join(:school_infos, id: :school_info_id).
+      join(:schools, id: :school_id).
+      paged_each(rows_per_fetch: 10_000) do |submission|
```

Comment:
is our dataset safe for this?  http://www.rubydoc.info/github/jeremyevans/sequel/Sequel%2FDataset%3Apaged_each

Reply:
I think so. The id columns should give an unambiguous order to the rows. The script was already using paged_each when querying FORMS so I used the same thing here.